### PR TITLE
Adding new Participants

### PR DESCRIPTION
**Pull request by Destroyer675000**
This is Destroyer675000 and i am just sending a pull request.

### CODE REVIEW

File: scripts/people/Destroyer675000.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Hello all.  I am Destroyer675000
+- blog: http://coreyflickinger.wordpress.com/
+  feed: http://coreyflickinger.wordpress.com/feed/
+  forges:
+    - http://github.com/Destroyer675000
+  irc: Destroyer675000
+  name: Corey Flickinger
+  rit_dce: crf9284
+
```
Comment (decause) [inline scripts/people/Destroyer675000.yaml]: What is this thingamabobber here?

Comment (decause) [inline scripts/people/Destroyer675000.yaml]: Just remove this last empty line all together, and I'll merge.

**Comment by decause** [comment]
yes @ralphbean is right. Blank lines at the end of a file can cause problems, and even though a .yaml file is not python, it is good to observe PEP008 style-guidelines: http://www.python.org/dev/peps/pep-0008/

Paging @Destroyer675000 in case you need a direct ping to notice this.

**Comment by Destroyer675000** [comment]
@decause just so that you can see what is the matter with me not showing up

**Pull request closed without merging**